Do not use `path` until Rook/Ceph bug is fixed

aj-jester opened the pull request:
- Do not use `path` until Rook/Ceph bug is fixed.
- Remove ServiceAddress and IngressAddress from K8s docs because they've been deprecated.

dexhorthy commented on content/docs/kubernetes/packaging-an-application/volumes.md:
```diff
@@ -76,10 +76,14 @@ spec:
 
-Paths within the shared filesystem can be included in snapshots by adding them to the [backup](/docs/snapshots/kubernetes/) section of your yaml.
+{{< warning title="Operating Systems Compatibility" >}}
+Due to an open bug in Rook/Ceph, custom `path` for `flexVolume` should be avoided.
```
Want to add a link to the github issue?

dexhorthy commented on content/docs/kubernetes/packaging-an-application/volumes.md:
```diff
@@ -76,10 +76,14 @@ spec:
+{{< warning title="Operating Systems Compatibility" >}}
+Due to an open bug in Rook/Ceph, custom `path` for `flexVolume` should be avoided.
+
+If its ommitted the `path` defaults to root, but if you do need to use `path` it needs to be included in snapshots by adding them to the [backup](/docs/snapshots/kubernetes/) section of your yaml.
```
what do you think of saying 

> `path` defaults to `/` 

instead of 

> `path` default to root

dexhorthy commented on content/docs/kubernetes/packaging-an-application/yaml-format.md:
````diff
@@ -34,7 +34,7 @@ The properties section includes definitions of some optional (but recommended) a
 ```yaml
 properties:
-  app_url: http://{{repl ServiceAddress nginx }}
+  app_url: http://{{repl ConsoleSetting "tls.hostname" }}
````
Do you prefer ConsoleSetting or ConfigOption here? I think many other examples use ConfigOption

aj-jester replied:
Oh yah, I can use `ConfigOption`, the reason I used `tls.hostname` because I knew it would always exist (I mean yes it could be blank but that variable exists) but I didn't check if for example `ConfigOption "Name"` does not exist then would things go side ways or would AKA gracefully handle it?

dexhorthy replied:
Gotcha, yeah for some reason i thought this snippet would include the `config` section but looks like its a little short. I don't think we should include it, I think ConsoleSetting is probably a better bet.

dexhorthy commented on content/docs/kubernetes/packaging-an-application/template-functions.md:
````diff
@@ -489,29 +489,6 @@ func Namespace() string
-```yaml
-properties:
-  app_url: '{{repl IngressAddress "frontend" 80 }}'
-```
````
👏 ⚔️

aj-jester commented:
@dexhorthy Awesome, thanks for the CR, I addressed all the comments except one, the one I'm not sure about if there was ever a bug report for this specific issue 😕. We happened to run into it when we were looking through the logs while troubleshooting the issue. I'll try to see if there is an existing GH issue to link it here...

dexhorthy commented:
Sounds good, can always add it on later

aj-jester commented:
Closing this in favor of https://github.com/replicatedhq/help-center/pull/901

aj-jester commented:
To add a little more context, it seems the issue has to do with the dir not being created before hand. A fix has been deployed to AKA which I believe resolves the `path` issue.
